src/Items.java

```diff
@@ -0,0 +1,31 @@
+public class Items {
+
+    String name;
+    int quantity;
+    double price;
+    double tax;
+    boolean isTaxExempt;
+    boolean isImported;
+
+    public Items(String name, int quantity, double price, boolean isTaxExempt, boolean isImported ){
+        this.name = name;
+        this.quantity = quantity;
+        this.price = price;
+        this.tax = tax;
+        this.isTaxExempt = isTaxExempt;
+        this.isImported = isImported;
+
+        if(this.isTaxExempt && this.isImported){
```

Comment:
This needs to be refactored to use 1 computed tax rate. ex: if imported add .05 to the tax rate and so on.